### PR TITLE
v2.0.0 RC1 (idam-session-cookie hotfix)

### PR DESCRIPTION
**Pull request by henrydobson**
### Change description ###

v2.0.0 RC1 (idam-session-cookie hotfix)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -150,7 +150,7 @@ allprojects  {
 
   task codeceptFunctional(type: Exec, dependsOn: [':yarnInstall', ':notifyClientInstall']) {
     workingDir '.'
-    commandLine 'node_modules/codeceptjs/bin/codecept.js', 'run', '--grep', '@functional', '--verbose', '--reporter', 'mocha-multi'
+    commandLine 'node_modules/codeceptjs/bin/codecept.js', 'run', '--grep', '@smoke', '--verbose', '--reporter', 'mocha-multi'
```
Comment (henrydobson) [inline build.gradle]:
```suggestion
    commandLine 'node_modules/codeceptjs/bin/codecept.js', 'run', '--grep', '@functional', '--verbose', '--reporter', 'mocha-multi'
```

File: security.sh [hunk 2]
```diff
@@ -21,4 +21,5 @@ while !(curl -s http://0.0.0.0:1001) > /dev/null
   cp *.html functional-output/
   zap-cli -p 1001 alerts -l Informational
   zap-cli --zap-url http://0.0.0.0 -p 1001 alerts -l High --exit-code False
-  curl --fail http://0.0.0.0:1001/OTHER/core/other/jsonreport/?formMethod=GET --output report.json
\ No newline at end of file
+
```
Comment (henrydobson) [inline security.sh]:
```suggestion
```

**Comment by henrydobson** [comment]
Can @shravanmechineni review the waits that have appeared in the tests?

**Comment by shravanmechineni** [comment]
> Can @shravanmechineni review the waits that have appeared in the tests?

Those waits can be removed.

File: src/test/js/self_registration_test.js [hunk 3]
```diff
@@ -155,6 +158,7 @@ Scenario('@functional @selfregister @prePopulatedScreen I can self register with
     I.fillField('#password2', TestData.PASSWORD);
     I.click('Continue');
     I.waitForText('Account created', 20, 'h1');
+    I.wait(5);
```
Comment (henrydobson) [inline src/test/js/self_registration_test.js]:
```suggestion
```

File: src/test/js/self_registration_test.js [hunk 4]
```diff
@@ -117,6 +119,7 @@ Scenario('@functional @selfregister I can self register', async (I) => {
     I.fillField('#password', TestData.PASSWORD);
     I.interceptRequestsAfterSignin();
     I.click('Sign in');
+    I.wait(10);
```
Comment (henrydobson) [inline src/test/js/self_registration_test.js]:
```suggestion
```

File: src/test/js/self_registration_test.js [hunk 5]
```diff
@@ -109,6 +110,7 @@ Scenario('@functional @selfregister I can self register', async (I) => {
     I.fillField('#password2', TestData.PASSWORD);
     I.click('Continue');
     I.waitForText('Account created', 20, 'h1');
+    I.wait(5);
```
Comment (henrydobson) [inline src/test/js/self_registration_test.js]:
```suggestion
```

File: src/test/js/self_registration_test.js [hunk 6]
```diff
@@ -36,6 +36,7 @@ Scenario('@functional @selfregister User Validation errors', (I) => {
     I.see('Create an account');
     I.click("Continue");
     I.waitForText('Information is missing or invalid', 20, 'h2');
+    I.wait(5);
```
Comment (henrydobson) [inline src/test/js/self_registration_test.js]:
```suggestion
```

**Comment by henrydobson** [comment]
Only waiting on https://build.platform.hmcts.net/blue/organizations/jenkins/HMCTS_IDAM%2Fidam-web-public/detail/PR-330/7/pipeline 

The latest change and triggered build are for a ci blocker.

**Pull request merged**